packages/harpguru-core/src/components/option-stack/components/list-stack/list-stack.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import { List } from '../list'
+import type {
+  OptionStackProps,
+  WithTransition,
+  OptionProps_Scales,
+  OptionProps_Apparatus,
+} from '../../types'
+import { isOptionProps_Scales } from '../../types'
+import { useInterpolateTransitionValue } from '../../hooks'
+
+export const ListStack = ({
+  stackPropsz,
+  transitionValue,
+}: OptionStackProps & WithTransition): React.ReactElement => {
+  const listStack = stackPropsz.map((items, index, array) => {
```

Comment:
Rename `items` to `optionProps` and `i` to `scaleOptionProps` or `apparatusOptionProps`, depending

Also consider renaming `stackPropsz` to `stackProps` across the board. I don't think that flagged additional plurality is actually meaningful now. It's already plural enough.

Reply:
All renamed